compatibility/bazel_tools/daml_script/daml_script.bzl

```diff
@@ -66,43 +70,80 @@ def daml_script_test(compiler_version, runner_version):
         server = "@daml-sdk-{version}//:sandbox-on-x".format(version = runner_version)
         server_args = ["--participant", "participant-id=sandbox,port=6865"]
         server_files = []
-        runner_files = ["$(rootpath {})".format(compiled_dar)]
+        server_files_prefix = ""
     else:
         server = daml_runner
         server_args = ["sandbox"]
         server_files = ["$(rootpath {})".format(compiled_dar)]
-        runner_files = []
+        server_files_prefix = "--dar=" if use_canton else ""
 
-    client_server_test(
-        name = "daml-script-test-compiler-{compiler_version}-runner-{runner_version}".format(
-            compiler_version = version_to_name(compiler_version),
-            runner_version = version_to_name(runner_version),
+    native.genrule(
+        name = "{}-client-sh".format(name),
+        outs = ["{}-client.sh".format(name)],
+        cmd = """\
+cat >$(OUTS) <<'EOF'
+#!/usr/bin/env bash
+set -euo pipefail
+canonicalize_rlocation() {{
+  # Note (MK): This is a fun one: Let's say $$TEST_WORKSPACE is "compatibility"
```

Comment:
The amount of questionable comments that I wrote months ago that keep resurfacing now is somewhat disturbing